README.md

```diff
@@ -350,6 +350,7 @@ supported:
   [no floating versions / ranges](https://github.com/oss-review-toolkit/ort/pull/1303#issue-253860146),
```

Comment:
Commit message nit: Please break the detailed message after 80 characters as it is described in the [CONTRIBUTING.md](https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md).